packages/ketcher-core/src/application/render/renderers/BondRenderer.ts

```diff
@@ -628,6 +639,18 @@ export class BondRenderer extends BaseRenderer {
         );
 
         this.pathShape = `M${startPosition.x},${startPosition.y} L${endPosition.x},${endPosition.y}`;
+        this.rootElement
+          .append('path')
+          .attr('d', this.pathShape)
+          .attr('stroke', 'black')
+          .attr('stroke-width', `${BOND_WIDTH * 10}px`)
+          .attr('opacity', '0')
+          .on('mouseenter', () => {
+            this.appendHover();
+          })
+          .on('mouseleave', () => {
+            this.removeHover();
+          });
```

Comment:
We have duplicated logic; it would be great to extract it into a private method, something like this:
```js
  private createPath(
    attrs: Record<string, string>,
    isHoverable: boolean,
  ): void {
    if (!this.rootElement) {
      return;
    }
    const path = this.rootElement.append('path').attr('d', this.pathShape);

    Object.entries(attrs).forEach(([key, value]) => {
      path.attr(key, value);
    });

    if (isHoverable) {
      path
        .on('mouseover', () => {
          this.appendHover();
        })
        .on('mouseout', () => {
          this.removeHover();
        });
    }
  }
```

And use it like this:
```js
        this.createPath(
          {
            stroke: 'transparent',
            'stroke-width': `${BOND_WIDTH * 10}px`,
            opacity: '0',
          },
          true,
        );

        this.createPath(
          {
            stroke: 'black',
            'stroke-width': `${BOND_WIDTH}px`,
          },
          true,
        );
```